data/linelists/GES/repackage_GES_linelist.jl

```diff
@@ -0,0 +1,72 @@
+using Korg, HDF5, FITSIO
+
+# this should point to the base dir of the Korg_data repo
+# https://github.com/ajwheeler/Korg_data
+Korg_data_dir = "../../../../Korg_data"
```

Comment:
Feel free to ignore this!
```suggestion
Korg_data_dir = joinpath(@__DIR__,"../../../../Korg_data")
```

Reply:
Thanks!